docs/content/using-npm/registry.md

```diff
@@ -30,6 +30,14 @@ The registry URL used is determined by the scope of the package (see
 supplied by the `registry` config parameter.  See [`npm config`](/commands/npm-config),
 [`npmrc`](/configuring-npm/npmrc), and [`config`](/using-npm/config) for more on managing npm's configuration.
 
+When the default registry is used in a package-lock or shrinkwrap is has the
+special meaning of "the currently configured registry". If you create a lock
+file while using the default registry you can switch to another registry and
+npm will install packages from the new registry, but if you create a lock
+file while using a custom registry packages will be installed from that
+registry even after you change to another registry. To update the registry in a
+lock file you can remove the file or remove or modify the resolved key.
```

Comment:
The package-lock is not something we want folks editing, it's assumed that the cli "owns" this. As such we should not suggest modifying the key directly.

Reply:
Hows this? Or delete the sentence?

> You should delete the file if you would like to change the custom registry.

Reply:
Just delete "or modify the resolved key"

Reply:
Or perhaps yes just deleting the sentence for now. Best to add something more clarifying later than put something in now that's not right.